pms_pwa/controllers/web.py

```diff
@@ -14,21 +14,33 @@ def index(self, *args, **kw):
             request.session.uid
         ).has_group("pms_pwa.group_pms_property_user"):
             return http.local_redirect(
-                "/pms-panel", query=request.params, keep_hash=True
+                "/", query=request.params, keep_hash=True
             )
         return super(Home, self).index(*args, **kw)
 
     def _login_redirect(self, uid, redirect=None):
         if not redirect and not request.env["res.users"].sudo().browse(uid).has_group(
```

Comment:
Sabeis que puede estar ocurriendo aquí? Me está redireccionando siempre al panel de reservas aunque el usuario no sea del grupo "property_user" (esto aunque lo comento ahora ya pasaba desde el principio)

Reply:
Hay un not delante de la comprobación del has_group por lo que la redirección debería hacerla cuando el usuario no tiene el grupo.

Reply:
pfff.... jaja Gracias!